presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -44,6 +44,12 @@ Constructors
 
     Returns a geometry type linestring object from WKT representation.
 
+.. function:: ST_LineString(array<Point>) -> LineString
+
+    Returns a LineString formed from an array of points. Empty or null points are ignored.  If there are fewer than
+    two non-empty points are in the input array, an empty LineString will be returned.  The returned geometry may
```

Comment:
typo: extra `are` in `are fewer than two non-empty points are in the input array`